packages/utils/interactive-hooks/src/useViewCommandFocus.win32.ts

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
-import { findNodeHandle, UIManager } from 'react-native';
+import { findNodeHandle, UIManager, View } from 'react-native';
 import { IViewWin32 } from '@office-iss/react-native-win32';
 
 const setAndForwardRef = require('./setAndForwardRef');
 
-export type IFocusable = IViewWin32;
+export interface IFocusable extends View, IViewWin32 {}
```

Comment:
Could this just be IView from adapters?
I actually think there's no point to make this hook win32 and macOS only (all platforms should support imperative focus calls), so I wonder if we could just make it completely generic that way...

Reply:
Should be fine as the adapter